core/crypto/crypto_store.hpp

```diff
@@ -91,7 +94,7 @@ namespace kagome::crypto {
      * @param pk public key to look for
      * @return found key pair if exists
      */
-    virtual boost::optional<ED25519Keypair> findEd25519Keypair(
+    virtual outcome::result<ED25519Keypair> findEd25519Keypair(
```

Comment:
Actually optional was more logical. It is not an error if requested key does not exist